### PR TITLE
Add "Good First Issues" subsection to README.md

### PR DESCRIPTION
**Pull request by bnb**
This adds a new section that is intended to be maintained over time, sharing which repos actively support first time contributors with the "good first issue" label.

For reference, if a repo wants to be added to this list they should have a non-trivial number of open issues labeled "good first issue" and be willing to provide mentorship/support on getting first time contributors over the barrier.

### CODE REVIEW

**Comment by bnb** [comment]
Closes #5

**Review by 3imed-jaberi — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -36,6 +36,17 @@ If you're looking to contribute directly to Node.js, these guides will help move
...
+
+Here's a set of repositories within the project that have multiple good first issues open:
+
+- [nodejs/node](https://github.com/nodejs/node/labels/good%20first%20issue): the core project. There are many good first issues here. The easiest ones often get snatched up quickly, but we try our best to ensure there are always some.
+- [nodejs/nodejs.dev](https://github.com/nodejs/nodejs.dev/labels/good%20first%20issue): our effort to build a new website based off of React using Gatsby.js.
+- [nodejs/citgm](https://github.com/nodejs/citgm/labels/good%20first%20issue): Canary in the Gold Mine, or CITGM, is a tool Node.js uses to test changes to see if they'll impact the most used modules in the ecosystem. It's a highly impactful project and one that's a great place to contribute to.
+- [node/community-committee](https://github.com/nodejs/community-committee/labels/good%20first%20issue): The Community Committee is a top-level committee in Node.js that foucses on outward-facing projects. Various aspects of the committee and its' initiatives are incredibly beginner-friendly.
```
Comment (keywordnew) [inline README.md]:
```suggestion
- [node/community-committee](https://github.com/nodejs/community-committee/labels/good%20first%20issue): The Community Committee is a top-level committee in Node.js that foucses on outward-facing projects. Various aspects of the committee and its initiatives are incredibly beginner-friendly.
```

**Review by keywordnew — APPROVED** [review]
Thank you, LGTM, pending nit.

**Review by WaleedAshraf — APPROVED** [review]
LGTM

**Review by mhdawson — APPROVED** [review]
LGTM

**Pull request merged**